lib/metrics/statsd.js

```diff
@@ -3,10 +3,40 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var StatsD = require('node-statsd')
+var uaParser = require('ua-parser')
 
 var STATSD_PREFIX = 'fxa.auth.'
 var TIMING_SUFFIX = '.time'
 
+function getGenericTags(info) {
+  var tags
+  if (info.userAgent) {
+    var agent = uaParser.parse(info.userAgent)
+    if (agent) {
+      if (agent.ua) {
+        tags = [
+            'agent_ua_family:' + agent.ua.family, // -> "Safari"
+            'agent_ua_version:' + agent.ua.toVersionString(), // -> "5.0.1"
+            'agent_ua_version_major:' + agent.ua.major // -> "5"
+        ]
+      }
+
+      if (agent.os) {
+        if(!tags){
```

Comment:
I originally did that but it caused this test to fail because it was checking for tags to be undefined.

https://github.com/mozilla/fxa-auth-server/blob/master/test/local/statsd_tests.js#L54

I didn't want to modify existing test case but I could to make code a little cleaner?